### ISSUE TITLE
No music on new question on Levels 5-14 when Double Dip used

### ISSUE THREAD

**Issue by Pyscowicz** [Macronair/TheMillionaireGame#77]
Hi, I tried Double Dip lifeline and after that I got my answer right and went to next question. Problem is that there is no question music playing. I have made fork of your project on 2022 and I have same problem on my version too. On first questions It's working fine.

Thank you for your great software.

**Comment by Macronair** [comment]
Hi @Pyscowicz , which version of the software do you currently use?
In the latest version, there is a fix applied for this bug. If you are using an older version, try using the new one and see if that will fix the problem.
(Don't forget to make a back-up of the old installation in case something goes wrong). ;)

**Comment by Pyscowicz** [comment]
I tested it on Release 1.1.756. Temporary fix for me is that If I lock answer and after that undo it then question music will play. Also setting range for questions is not working but I noticed that It's mentioned on issues page.

**Comment by Macronair** [comment]
Hmm strange. I'm going to check where things are going wrong. But that will be after this weekend because of my busy schedule.

I expect these issues will be fixed (hopefully 😅) in the next version which will be 1.2.

Also one question: do you get some sort of error message when the sound cue is not playing?

Thanks for letting me know btw!

**Comment by Pyscowicz** [comment]
No, there is no any error messages on that situation, on your version or on my fork.

Thanks!

**Comment by Macronair** [comment]
Checked this once more for the new 1.2 update. I improved and hopefully fixed this issue.
